api-types.ts

```diff
@@ -1,131 +0,0 @@
-export type Maybe<T> = T | null;
```

Comment:
This file is out of date and isn't used anymore. The front end uses `frontend/src/types/api-types.ts` instead.